lib/utils/Thing.js

```diff
@@ -129,7 +134,7 @@ export default class Thing {
 	}
 
 	isComment() {
-		return this.thing.classList.contains('comment');
+		return this.thing.classList.contains('comment') || this.thing.classList.contains('was-comment');
```

Comment:
Looks like comments in your inbox previously would return false for both `isComment` and `isLink`, so this shouldn't break anything.

In the future maybe we should use the attr `data-type = link|comment|message`